core/src/test/java/com/oath/oak/OakViewTests.java

```diff
@@ -0,0 +1,194 @@
+    private static final int ELEMENTS = 1000;
+
+    @Before
+    public void init() {
+        OakMapBuilder<String, String> builder = new OakMapBuilder<String, String>()
+                .setChunkMaxItems(100)
+                .setChunkBytesPerItem(128)
+                .setKeySerializer(new StringSerializer())
+                .setValueSerializer(new StringSerializer())
+                .setComparator(new StringComparator())
+                .setMinKey("");
+
+        oak =  builder.build();
+
+        for (int i = 0; i < ELEMENTS; i++) {
+            String key = String.valueOf(i);
+            String val = String.valueOf(i);
+            oak.put(key, val);
+        }
+    }
+
+    @After
+    public void tearDown() {
+        oak.close();
+    }
+
+    @Test
+    public void uniTestOakRBuffer(){
+
+
+        Function<ByteBuffer, String> deserialize = (byteBuffer) -> {
+            int size = byteBuffer.getInt(0);
+            StringBuilder object = new StringBuilder(size);
+            for (int i = 0; i < size; i++) {
+                char c = byteBuffer.getChar(Integer.BYTES + byteBuffer.position() + i * Character.BYTES);
+                object.append(c);
+            }
+            return object.toString();
+        };
+
+        try(OakBufferView<String> oakBV = oak.createBufferView()) {
+            String testVal = String.valueOf(123);
+            ByteBuffer testValBB = ByteBuffer.allocate(Integer.BYTES + testVal.length()*Character.BYTES);
+            testValBB.putInt(0, testVal.length());
+            for (int i = 0; i< testVal.length(); ++i){
+                testValBB.putChar(Integer.BYTES + i*Character.BYTES, testVal.charAt(i));
+            }
+
+
+            OakRBuffer valBuffer = oakBV.get(testVal);
+            String transformed = valBuffer.transform(deserialize);
+            assertEquals(testVal, transformed);
+
+            assertEquals(testValBB.capacity(), valBuffer.capacity());
+
+            assertEquals(testVal.length(), valBuffer.getInt(0));
+            assertEquals(testValBB.getInt(1), valBuffer.getInt(1));
+
+            for(int i =0 ; i < testVal.length(); ++i) {
+                assertEquals(testVal.charAt(i), valBuffer.getChar(i*2 + Integer.BYTES));
+            }
+
+            byte[] testValBytes = testVal.getBytes();
+            for(int i =0 ; i < testValBytes.length; ++i) {
+                assertEquals(testValBytes[i], valBuffer.get(i*2+1 + Integer.BYTES));
+            }
+
+            assertEquals(testValBB.getDouble(1), valBuffer.getDouble(1));
+
+            assertEquals(testValBB.getFloat(1), valBuffer.getFloat(1));
+
+            for(int i =0 ; i < testValBytes.length/Short.BYTES; ++i) {
+                assertEquals(testValBB.getShort(i), valBuffer.getShort(i));
+            }
+
+            for(int i =0 ; i < testValBytes.length/Long.BYTES; ++i) {
+                assertEquals(testValBB.getLong(i), valBuffer.getLong(i));
+            }
+        }
+    }
+
+
+
+    @Test
```

Comment:
Can use this:
```suggestion
    @Test(expected = ConcurrentModificationException.class)
```

and remove the try-catch-fail logic below